metron-deployment/roles/kibana/README.md

```diff
@@ -1,35 +0,0 @@
-Kibana 4
---------
-
-This role installs Kibana along with the default Metron Dashboard.
-
-### FAQ
-
-#### How do I change Metron's default dashboard?
```

Comment:
How does the load of the Kibana Dashboard work now?  Do we not use `elasticdump` any longer?  Bravo, if you found a better way.

This README was intended to help describe how to make changes to the Metron Kibana Dashboard and get those changes into source control.  I just want to make sure this same sort of thing is either obvious now or described somewhere else.